vllm/worker/model_runner.py

```diff
@@ -32,12 +41,31 @@ def __init__(
         self.model = None
         self.block_size = None  # Set after initial profiling.
 
+        self.graph_runners: Dict[int, CUDAGraphRunner] = {}
```

Comment:
another way to save memory is to use LRUcache for this map, and capture it on demand.

Reply:
Made a new issue on this. Let's do it in a followup PR.